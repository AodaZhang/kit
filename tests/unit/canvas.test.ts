/**
 * @description canvas相关函数-单元测试
 * @author aodazhang 2021.02.04
 */
import { canvas } from '@/index'

describe('createTextImage', () => {
  it('1.快照调用', () => {
    expect(canvas.createTextImage).toMatchSnapshot()
  })

  it('2.入参非法', () => {
    expect(canvas.createTextImage(null)).toBe('')
    expect(canvas.createTextImage(undefined)).toBe('')
  })
})
