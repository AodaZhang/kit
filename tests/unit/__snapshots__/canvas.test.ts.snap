// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createTextImage 1.快照调用 1`] = `[Function]`;
